addons/medical_engine/script_macros_medical.hpp

```diff
@@ -1,3 +1,7 @@
+// #define DEBUG_MODE_FULL
```

Comment:
"Defined" in medical_engine/script_component.hpp

Reply:
I'd rather keep them in both places
script_macros_medical.hpp lets you enable for all 11 medical modules in one place
but you can still go for individual modules with script_component.hpp

a lot of times you have to balance too much vs too little debuging info